wrench/Cargo.toml

```diff
@@ -33,3 +33,7 @@ dwrote = "0.1.2"
 
 [target.'cfg(target_os = "linux")'.dependencies]
 font-loader = "0.1.0"
+
+[target.'cfg(any(target_os = "linux", target_os = "macos"))'.dependencies]
```

Comment:
given that we use it for automated tests only, can we hide this dependency behind a feature gate?